ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/monitor-telemetry-postcard-policy.md

```diff
@@ -0,0 +1,116 @@
+# monitor-telemetry-postcard-policy
+
+## Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+- [Monitoring](#monitoring)
+  - [Monitor Telemetry Postcard Policy](#monitor-telemetry-postcard-policy)
+
+## Management
+
+### Management Interfaces
+
+#### Management Interfaces Summary
+
+##### IPv4
+
+| Management Interface | Description | Type | VRF | IP Address | Gateway |
+| -------------------- | ----------- | ---- | --- | ---------- | ------- |
+| Management1 | oob_management | oob | MGMT | 10.73.255.122/24 | 10.73.255.2 |
+
+##### IPv6
+
+| Management Interface | Description | Type | VRF | IPv6 Address | IPv6 Gateway |
+| -------------------- | ----------- | ---- | --- | ------------ | ------------ |
+| Management1 | oob_management | oob | MGMT | - | - |
+
+#### Management Interfaces Device Configuration
+
+```eos
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+```
+
+## Monitoring
+
+### Monitor Telemetry Postcard Policy
+
+#### Sample Policy Summary
+
+##### samplepo1
+
+###### Match rules
+
+| Rule Name | Rule Type | Source Prefix | Destination Prefix | Protocol | Source Ports | Destination Ports |
+| --------- | --------- | ------------- | ------------------ | -------- | ------------ | ----------------- |
+| rule1 | ipv4 | 3.4.5.0/24 | 10.3.3.0/24 | tcp<br>udp | -<br>98 | 77, 78-80, www<br>99 |
+| rule2 | ipv6 | 5::0/128 | 4::0/128 | udp | - | 747, 748-800, ssh |
+| rule3 | ipv4 | - | - | - | - | - |
+
+##### samplepo2
+
+###### Match rules
+
+| Rule Name | Rule Type | Source Prefix | Destination Prefix | Protocol | Source Ports | Destination Ports |
+| --------- | --------- | ------------- | ------------------ | -------- | ------------ | ----------------- |
+| rule1 | ipv4 | 3.4.5.0/24 | 10.3.3.0/24 | udp | 77, 78-80, www | 88, www |
+
+#### Telemetry Postcard Policy Profiles
+
+| Profile Name | Ingress Sample Policy |
+| ------------ | --------------------- |
+| profile1 | samplepo1 |
+| profile2 | samplepo2 |
+
+#### Monitor Telemetry Postcard Configuration
+
+```eos
+!
+monitor telemetry postcard policy
+   no disabled
+   ingress sample rate 16384
+   marker vxlan header word 0 bit 0
+   ingress collection gre source 10.3.3.3 destination 10.3.3.4 version 2
+   !
+   sample policy samplepo1
+      match rule1 ipv4
+         destination prefix 10.3.3.0/24
+         source prefix 3.4.5.0/24
+         protocol tcp destination port 77
+         protocol tcp destination port 78-80
+         protocol tcp destination port www
+         protocol udp destination port 99
+         protocol udp source port 98
```

Comment:
protocol ports comes like this 
```
protocol tcp destination port 77-80, 90
protocol udp source port 98 destination port 22, 99, 747-800
```

Reply:
if we add port number and port names in a single command, eos is giving invalid input error.

s1-leaf1(config-postcard-sample-policy-match-samplepo1-rule1-ipv4)#protocol tcp destination port 77, 78-80, www
% Invalid input